lab/container-images/ipa-images/bootfs/Dockerfile

```diff
@@ -61,14 +64,22 @@ RUN apk add --root alpine-minirootfs \
 RUN apk add --root alpine-minirootfs \
 	gcc \
 	python3-dev \
-	musl-dev
+	musl-dev \
+	linux-headers
 
 # pip can't download packages when it runs in chroot for some reasons.
-# TODO: try to insall everything here and then just copy it to the destination image.
 RUN mkdir alpine-minirootfs/tmp/pip3
 COPY requirements.txt alpine-minirootfs/tmp/pip3/
 RUN pip3 download -r alpine-minirootfs/tmp/pip3/requirements.txt --no-deps -d alpine-minirootfs/tmp/pip3/
-RUN chroot alpine-minirootfs pip3 install --isolated --no-index --find-links /tmp/pip3 -r /tmp/pip3/requirements.txt
+RUN <<EOF
+chroot alpine-minirootfs sh -e -c "
+mkdir /usr/lib/ironic-python-agent
+cd /usr/lib/ironic-python-agent
+python3 -m venv .
```

Comment:
Why do you need venv for this case? It's not multi-user environment, no different apps with different requirements etc. Why not just continue installing it into the system?

Reply:
simplified, but also changed some other stuff to make it even simpler